### PR TITLE
Updates to FreeBSD preparation

### PR DESCRIPTION
**Pull request by mattias-p**
Adds:
1. Updating of the ports database
2. Installation of curl (a prerequisite for the backend sanity check instruction)

### CODE REVIEW

File: docs/internal-documentation/distrib-testing/FreeBSD-Preparation.md [hunk 1]
```diff
@@ -18,10 +18,11 @@
    visudo
```
Comment (matsduf) [inline docs/internal-documentation/distrib-testing/FreeBSD-Preparation.md]: Either sudo has to be installed, or use good old su.

**Comment by mattias-p** [comment]
I believe you're already root at this point. It's a clean installation.

File: docs/internal-documentation/distrib-testing/FreeBSD-Preparation.md (hunk 1 above)
Comment (mattias-p) [inline docs/internal-documentation/distrib-testing/FreeBSD-Preparation.md]: Good point. I'm opting to install `sudo`. This way it's easier to have common instructions across operating systems for system testing.

**Comment by matsduf** [comment]
All steps in this document assumes that you are root. I guess it is OK that it is implicitly so for now.

**Pull request merged**